src/components/discover-sheet/RegisterENSSection.tsx

```diff
@@ -28,7 +30,9 @@ export default function RegisterENSSection() {
 
   const handlePress = useCallback(() => {
     if (!isReadOnlyWallet || enableActionsOnReadOnlyWallet) {
-      navigate(Routes.REGISTER_ENS_NAVIGATOR);
+      navigate(Routes.REGISTER_ENS_NAVIGATOR, {
+        fromDiscover: true,
```

Comment:
Is this being used anywhere?

Reply:
it prevents the status bar from changing to the wrong color in light mode when you dismiss a sheet that's on top of another sheet

check out https://github.com/rainbow-me/rainbow/blob/develop/src/navigation/nativeStackConfig.js#L23-L30

status bar handling needs a refactor so just a temporary fix till we find time, although open to cleaner solutions if you've got ideas

Reply:
Ahh gotcha! All good then!